src/amp/lib/permutive.ts

```diff
@@ -0,0 +1,39 @@
+export interface PermutivePayload {
+    'properties.content.Premium'?: string;
+    'properties.content.id'?: string;
+    'properties.content.title'?: string;
+    'properties.content.section'?: string;
+    'properties.content.authors!'?: string;
+    'properties.content.keywords!'?: string;
+    'properties.content.publishedAt'?: string;
+    'properties.user.edition'?: string;
+}
+
+export const generatePermutivePayload = (
+    rawConfig: CommercialConfigType,
+): PermutivePayload => {
+    const publishedAt =
+        rawConfig.webPublicationDate &&
+        typeof rawConfig.webPublicationDate === 'number'
+            ? new Date(rawConfig.webPublicationDate).toISOString()
+            : void 0;
+    const config: { [key: string]: any } = {
+        'properties.content.Premium': rawConfig.isPaidContent,
+        'properties.content.id': rawConfig.pageId,
+        'properties.content.title': rawConfig.headline,
+        'properties.content.section': rawConfig.section,
+        'properties.content.authors!list[string]': rawConfig.author,
+        'properties.content.keywords!list[string]': rawConfig.keywords,
```

Comment:
Dp we need to make sure these fields are properly formatted comma separated strings?

Reply:
In an ideal world, the `author` (`authors?) and `keywords` properties would be arrays of strings, as `string[]` and could then be `join`'d using a comma

Not sure if this is relevant or not but elsewhere in DCR we pull the author string(s) out of `CAPI.tags` using 

```
const contributorTags = tags.filter(tag => tag.type === 'Contributor');
```

and then get the author string from `tag.title`

Reply:
It's the AMP/permutive spec that expects comma separated strings for both authors and keywords, which is how it is already in the config store. But I will strip the white spaces explicitly to avoid any surprises :)